src/Files.App/UserControls/Sidebar/SidebarStyles.xaml

```diff
@@ -158,15 +162,7 @@
 								</AnimatedIcon.RenderTransform>
 							</AnimatedIcon>
 
-							<FontIcon
-								x:Name="ExpanderIcon"
-								Grid.Column="2"
-								Margin="8,0,14,0"
-								AutomationProperties.AccessibilityView="Raw"
-								FontSize="8"
-								Glyph="&#xE70E;"
-								Visibility="Collapsed" />
-
```

Comment:
Was this removed by mistake?

Reply:
This is not used. I've ensured in removing.
I think this was created in a dev stage.